digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -1832,6 +1832,14 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     }
   }
 
+  get("/getAssetTypes") {
+    AssetTypeInfo.values.filterNot(_.typeId == 99).toList.sortBy(_.typeId).map { asset =>
```

Comment:
instead of '99' you have on asset.scala an UnknownAssetTypeId that extends AssetTypeInfo with typeId = 99

Reply:
Done.